test/controllers/api/v1/contacts_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class Api::V1::ContactsControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
No backend functionality is required in this challenge. Please remove this file.

Reply:
Removed the file. Actually I was trying to try out making api functionality for contacts, just for learning purposes, but decided against it. And forgot that I should be deleting these additional files.

Reply:
That's fine. But you can continue with making the full fledged version of this project once you finish up this challenge.

Reply:
Yes. It will help revise the LRRB concepts 😅